app/controllers/orga/conferences_controller.rb

```diff
@@ -35,9 +38,9 @@ def conference
 
   def conference_params
     params[:conference] ? params.require(:conference).permit(
```

Comment:
You usually know it when you remove it, run the tests and everything still works 😉  But yeah, risky, let's just focus on what you actually want to change.

I also had a quick look at the whole controller code. There's this line:
```ruby
@conference ||= params[:id] ? Conference.find(params[:id]) : Conference.new(conference_params)
```
This for instance causes looking for a non-existing conference to just create a new one instead of raising an error (Record not Found) - so that's why the hack with the empty hash exists 💡